### PR TITLE
Update Main Menu Tests

### PR DESCRIPTION
**Pull request by ferdinh**
## Description
This PR updates Main Menu Tests.

It adds two test:
- Test that ensure `Panels` in main menu is setup correctly for the first time.
- Test that ensure `Panels` is toggled correctly.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Code Refactor (styling fix, extracting methods, etc. Does not cause any functionality changes)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: DEFCON Z/Assets/Tests/PlayMode Tests/MainMenuTest.cs [hunk 1]
```diff
@@ -1,56 +1,116 @@
...
-            Assert.That(menuTest.menuPanel.activeSelf, Is.EqualTo(true));
+            // Arrange
+            bool expectedMenuPanelState = true;
+            bool expectedSettingsPanelState = false;
+
+            // Act
+            // This part has been initiated in the Setup process.
+
+            // Assert
+            Assert.That(MainMenu.menuPanel.activeSelf, Is.EqualTo(expectedMenuPanelState));
```
Comment (AlexDyre) [inline DEFCON Z/Assets/Tests/PlayMode Tests/MainMenuTest.cs]: Second Assert should be moved to a separate test.
Without using multiple assert, the second Assert (should) never be able to be tested.

Comment (ferdinh) [reply DEFCON Z/Assets/Tests/PlayMode Tests/MainMenuTest.cs]: I'll create two new tests. First test will test whether menu panel state should be in the correct state and another one tests whether main settings panel is at the correct state when first initialized?

Comment (AlexDyre) [reply DEFCON Z/Assets/Tests/PlayMode Tests/MainMenuTest.cs]: That sounds about right yeah. One should be true the other false. Could always Assert that (menu && !settings) too.

Comment (ferdinh) [reply DEFCON Z/Assets/Tests/PlayMode Tests/MainMenuTest.cs]: Okay sweet, thank you.

**Comment by ferdinh** [comment]
Code updated @AlexDyre.

**Comment by ferdinh** [comment]
Thank you @AlexDyre.

**Pull request merged**